### ISSUE TITLE
Can't open .NCGR and .NCBR files

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Barubary/tiledggd#4]
```
This is a very strange problem. I've to do a little introduction.

Some time ago, I changed my pc with a new one, which mounts Windows7, and
re-downloaded the program from the website.
Before that, I used to open every type of file on my old computer.

I've realized it only in this moment: on the new pc, TGDD doesn't open
graphic NDS files like .NCGR and .NCBR anymore!
I tried to open the program in Administrator mode, but nothing changes. I
also tried to open the files sliding or using the File menu: in the first
case, the program simply does nothing; in the second case, the program
shows an error message (it's in italian, but it should be understandable):

"Impossibile caricare il file o l'assembly 'LuaInterface,
Version=2.0.0.16708, Culture=neutral, PublicKeyToken=null' o una delle
relative dipendenze. Tentativo di caricare un programma con un formato non
corretto."

Basically, it says that the file format is not correct... But how could it
be?! The program shouldn't pay attention to the actual "file format", as it
opens everything!

Indeed, the strange thing is that it opens EVERY file format but these!

I don't know if it depends on the operating system, because the program
works pretty well... I tried to run TGDD in compatibility mode (XP SP3),
and also reloading the bindings, but the problem remains.
As I said, on the old pc this problem doesn't exists...
I don't know what to think.

Thanks.
```

Original issue reported on code.google.com by `francesc...@gmail.com` on 9 Jan 2010 at 7:38
